app/src/androidTest/resources/reference_tests/domain_matching_tests.json

```diff
@@ -53,10 +53,7 @@
                 "siteURL": "https://ignore.test/",
                 "requestURL": "https://bad.third-party.site/",
                 "requestType": "script",
-                "expectAction": "block",
-                "exceptPlatforms": [
-                    "android-browser"
-                ]
+                "expectAction": "block"
```

Comment:
Remember to open a PR against the ref tests repo :)

Reply:
PR opened: https://github.com/duckduckgo/privacy-reference-tests/pull/73